src/hooks/nostr/useUserModeration.js

```
@@ -0,0 +1,235 @@
+      };
+
+      const subscription = ndk.subscribe(filter, {
+        closeOnEose: false,
+        cacheUsage: NDKSubscriptionCacheUsage.CACHE_FIRST,
+      });
+
+      subscription.on('event', (event) => {
+        try {
+          // Extract the user pubkey being muted from the 'p' tag
+          const userPubkeyTag = event.tags.find(tag => tag[0] === 'p');
+          if (userPubkeyTag && userPubkeyTag[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)